modules/installation-osp-custom-subnet.adoc

```diff
@@ -0,0 +1,32 @@
+// Module included in the following assemblies:
+//
+
+// * installing/installing_openstack/installing-openstack-installer-custom.adoc
+// * installing/installing_openstack/installing-openstack-installer-kuryr.adoc
+// * installing/installing_openstack/installing-openstack-user.adoc
+// * installing/installing_openstack/installing-openstack-user-kuryr.adoc
+
+[id="installation-osp-custom-subnet_{context}"]
+= Custom subnets in {rh-openstack} deployments
+
+Optionally, you can deploy a cluster on a {rh-openstack-first} subnet of your choice. The subnet's GUID is passed as the value of `platform.openstack.machinesSubnet` in the `install-config.yaml` file.
+
+This subnet is used as the cluster's primary subnet; nodes and ports are created on it.
+
+Before you run the {product-title} installer with a custom subnet, verify that:
+
+* The target network and subnet are available.
+
+* DHCP is enabled on the target subnet.
+
+* You can provide credentials to the installer that have permission to create ports on the target network.
```

Comment:
_You can provide installer credentials that have permission_? If that still works, it's a little easier for me to read.

Reply:
I think that's probably fine.